python/spec/dependabot/python/requirement_spec.rb

```diff
@@ -116,6 +116,17 @@
       end
     end
 
+    context "with multiple operators after the first" do
+      let(:requirement_string) { ">=2.0<2.1<2.2" }
+      # Python ignores operators after the first!
+      it { is_expected.to eq(Gem::Requirement.new(">=2.0")) }
```

Comment:
Curious why you added a new test case rather than modifying the existing test case from https://github.com/dependabot/dependabot-core/commit/45f5b77a776cf8bada4c262af9e39ff492ab6ba9#diff-606e746adf06604fa9039dd903280cffd0c3ee1485e2d54ad5e13932279a019aR40-R44 ?

Is there a scenario you envision where it's useful to test 2 operators that wouldn't be caught by testing 3 operators?

Reply:
Normally I prefer adding separate specs over modifying existing ones, mostly because I don't want to lose any coverage and many times I don't feel I can tell for sure whether that would be the case if I modify the spec. I also think it makes patches more focused, although it normally introduces some duplication and overhead. In this case it would probably be fine to change the existing spec, although it would be a bit weird to only test three operators instead of two? I think testing both cases illustrates that there's a bit more complexity when handling more than two.

Reply:
Sounds reasonable.